blockchain/chain_sync/src/sync_worker.rs

```diff
@@ -445,8 +445,8 @@ where
             .map_err(|e| (*block_cid, e.into()))?;
 
         // Retrieve lookback tipset for validation.
-        let lbts = cs
-            .get_lookback_tipset_for_round(base_ts.clone(), block.header().epoch())
+        let (lbts, _) = sm
```

Comment:
can get rid of the tipset state retrieval below if you just get it from here (Tipset state isn't calculated twice, but can still avoid the atomic access through mutex)